specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2021-02-01-preview/redisenterprise.json

```diff
@@ -0,0 +1,1762 @@
+      "properties": {
+        "clientProtocol": {
+          "type": "string",
+          "description": "Specifies whether redis clients can connect using TLS-encrypted or plaintext redis protocols. Default is TLS-encrypted.",
+          "enum": [
+            "Encrypted",
+            "Plaintext"
+          ],
+          "x-ms-enum": {
+            "name": "Protocol",
+            "modelAsString": true
+          }
+        },
+        "port": {
+          "type": "integer",
+          "format": "int32",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
+          "description": "TCP port of the database endpoint. Specified at create time. Defaults to an available port."
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ProvisioningState",
+          "readOnly": true,
+          "description": "Current provisioning status of the database"
+        },
+        "resourceState": {
+          "$ref": "#/definitions/ResourceState",
+          "readOnly": true,
+          "description": "Current resource status of the database"
+        },
+        "clusteringPolicy": {
+          "type": "string",
+          "description": "Clustering policy - default is OSSCluster. Specified at create time.",
+          "enum": [
+            "EnterpriseCluster",
+            "OSSCluster"
+          ],
+          "x-ms-enum": {
+            "name": "ClusteringPolicy",
+            "modelAsString": true
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "evictionPolicy": {
+          "type": "string",
+          "description": "Redis eviction policy - default is VolatileLRU",
+          "enum": [
+            "AllKeysLFU",
+            "AllKeysLRU",
+            "AllKeysRandom",
+            "VolatileLRU",
+            "VolatileLFU",
+            "VolatileTTL",
+            "VolatileRandom",
+            "NoEviction"
+          ],
+          "x-ms-enum": {
+            "name": "EvictionPolicy",
+            "modelAsString": true
+          }
+        },
+        "persistence": {
+          "$ref": "#/definitions/Persistence",
+          "description": "Persistence settings"
+        },
+        "modules": {
+          "type": "array",
+          "description": "Optional set of redis modules to enable in this database - modules can only be added at creation time.",
+          "items": {
+            "$ref": "#/definitions/Module"
+          },
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "georeplication": {
+          "type": "object",
+          "description": "Optional set of properties to configure geo replication for this database.",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
+          "properties": {
+            "georeplicationGroupNickname": {
```

Comment:
listing georeplication in this name again is redundant if this is a property of geoReplication.  For example, see how we have property 'name' under 'sku' rather than using 'skuName'.

Reply:
This change sounds good to me. We can go with just 'groupNickname'. @RamyasreeChakka what do you think?

Reply:
sounds good to me.